examples/notebooks/protein_ligand_complex_notebook.ipynb

```diff
@@ -75,21 +75,19 @@
   {
    "cell_type": "code",
    "execution_count": 2,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Warning: No xgboost installed on your system\n",
-      "Attempting to run %s will throw runtime errors\n"
-     ]
-    }
-   ],
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
    "source": [
     "import deepchem as dc\n",
+    "from deepchem.utils import download_url\n",
```

Comment:
This is the difference.  I didn't want to check in a 20+ mb blob so I lazily download it from S3.  The dataframe is csv.gzip now so can load in python 2.7 or 3.5.